### PR TITLE
add docs of ceph-csi

### PR DESCRIPTION
**Pull request by min-zh**
Signed-off-by: zhangmin <arminzhang@yunify.com>

### CODE REVIEW

File: content/en/docs/installing-on-linux/introduction/storage-configuration.md [hunk 1]
```diff
@@ -53,8 +53,9 @@ For more chart values, see [configuration](https://github.com/kubesphere/helm-ch
 
 # Ceph RBD
```
Comment (FeynmanZhou) [inline content/en/docs/installing-on-linux/introduction/storage-configuration.md:54]: Replace all second-level headings with two `##`, take an example:

`# Ceph RBD` => `## Ceph RBD`

File: content/en/docs/installing-on-linux/introduction/storage-configuration.md [hunk 2]
```diff
@@ -69,7 +70,74 @@ Here is an example of rbd-provisioner.
       - ceph.userKey=SHOULD_BE_REPLACED
       - sc.isDefault=true
 ```
-For more values, see [configuration](https://github.com/kubesphere/helm-charts/tree/master/src/test/rbd-provisioner#configuration))
+For more values, see [configuration](https://github.com/kubesphere/helm-charts/tree/master/src/test/rbd-provisioner#configuration)
+
+# Ceph CSI
```
Comment (FeynmanZhou) [inline content/en/docs/installing-on-linux/introduction/storage-configuration.md:75]: Replace all second-level headings with two ##

File: content/en/docs/installing-on-linux/introduction/storage-configuration.md [hunk 3]
```diff
@@ -69,7 +70,74 @@ Here is an example of rbd-provisioner.
...
+- name: ceph-csi-rbd-sc
+  sources:
+    yaml:
+      path:
+      - /root/ceph-csi-rbd-sc.yaml
+```
+For more information, see [chart for ceph-csi-rbd](https://github.com/ceph/ceph-csi/tree/master/charts/ceph-csi-rbd)
+
 
 # Glusterfs
```
Comment (FeynmanZhou) [inline content/en/docs/installing-on-linux/introduction/storage-configuration.md:142]: Replace all second-level headings with two `##`

File: content/en/docs/installing-on-linux/introduction/storage-configuration.md (hunk 1 above)
Comment (min-zh) [reply content/en/docs/installing-on-linux/introduction/storage-configuration.md:54]: ok

File: content/en/docs/installing-on-linux/introduction/storage-configuration.md (hunk 2 above)
Comment (min-zh) [reply content/en/docs/installing-on-linux/introduction/storage-configuration.md:75]: ok

File: content/en/docs/installing-on-linux/introduction/storage-configuration.md (hunk 3 above)
Comment (min-zh) [reply content/en/docs/installing-on-linux/introduction/storage-configuration.md:142]: ok

**Pull request merged**